src/cljs/main/broadfcui/page/workspace/data/tab.cljs

```diff
@@ -218,10 +220,9 @@
                                                         [DataImporter {:workspace-id workspace-id
                                                                        :this-realm this-realm
                                                                        :import-type "data"
-                                                                       :reload
-                                                                       (fn [entity-type]
-                                                                         ((:request-refresh props))
-                                                                         (react/call :refresh (@refs "entity-table") entity-type))}])}]]]))
+                                                                       :do-this-afterwards
+                                                                       (fn []
+                                                                         (react/call :refresh (@refs "entity-table") (:selected-entity-type @state)))}])}]]]))
```

Comment:
Ah yes.  In cases like this, it's preferable for the outer function to be explicit, and the inner one to be abbreviated.  Or you can just leave them both explicit, it doesn't really matter.